andela/ah-nyati-frontend
# #167756400 Implement article deletion by author (article)

#### What does this PR do?
- Implements the feature that enables an author to delete his articles

#### Description of Task to be completed?
- Create delete button
- Create delete action
- Write unit tests


#### How should this be manually tested?
- Log in to the app using username: **john.doe@andela.com** and password: **password**
- On the app landing page, navigate to the articles page and click on the last page number **91** or **92** and click on the last article with title **Article marked for extermination** to view it.
- Click on the delete button at the top right corner above the article image. A delete confirmation modal will appear.
- Click on delete. A success message will appear to show that delete was successful.

#### Any background context you want to provide?
- 

#### What are the relevant pivotal tracker stories?
- [#167756400](https://www.pivotaltracker.com/story/show/167756400)


#### Screenshots (if appropriate)
-

#### Questions:
-
